api-report/sequence.api.md

```diff
@@ -133,11 +133,13 @@ export interface IIntervalCollectionEvent<TInterval extends ISerializableInterva
     (event: "propertyChanged", listener: (interval: TInterval, propertyDeltas: PropertySet, local: boolean, op: ISequencedDocumentMessage | undefined) => void): any;
 }
 
-// @public (undocumented)
+// @public @sealed (undocumented)
```

Comment:
i'd expect more additions to the public API than this. notably:

- the sequence interval helpers, typed as an IIntervalHelpers
- a creation function for each of the start-in-range and end-in-range indexes

that way, people can actually use them :)